src/react/components/common/assetPreview/tfrecordAsset.test.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { IAssetProps } from "./assetPreview";
+import { ReactWrapper, mount } from "enzyme";
+import { TFRecordAsset } from "./tfrecordAsset";
+import MockFactory from "../../../../common/mockFactory";
+import { TFRecordsBuilder, FeatureType } from "../../../../providers/export/tensorFlowRecords/tensorFlowBuilder";
+import HtmlFileReader from "../../../../common/htmlFileReader";
+
+describe("Image Asset Component", () => {
+    // tslint:disable-next-line:max-line-length
+    const dataImage64 = "R0lGODlhEAAQAMQAAORHHOVSKudfOulrSOp3WOyDZu6QdvCchPGolfO0o/XBs/fNwfjZ0frl3/zy7////wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAkAABAALAAAAAAQABAAAAVVICSOZGlCQAosJ6mu7fiyZeKqNKToQGDsM8hBADgUXoGAiqhSvp5QAnQKGIgUhwFUYLCVDFCrKUE1lBavAViFIDlTImbKC5Gm2hB0SlBCBMQiB0UjIQA7";
+    const dataImage = new Uint8Array(Buffer.from(dataImage64, "base64"));
+
+    // tslint:disable-next-line:max-line-length
```

Comment:
don't think this is needed anymore

Reply:
Unfortunately I still see lint error without it.

Reply:
good catch tks